### DISCUSSION TITLE
Model versioning with existing unsuffixed model

### DISCUSSION THREAD

**Discussion by cyclonstep** [dbt-labs/dbt-core#10416] [Q&A] [answered]
Hello community! 
Recently I have this situations where I need to do breaking changes to the existing models of our current important models. 
Because this model is quite important for our business process, I want to do the versioning that based on this feature. https://docs.getdbt.com/docs/collaborate/govern/model-versions

When I read this, the pattern is I need to add suffixes to both the v1 (old one) and v2 (new one). If my understanding was not mistaken, this would also create the new models for both model_v1 and model_v2 (i want to avoid the creation the new v1 model, because this incremental model will be quite expensive). Is there anyway that I can use the old model for the v1? Currently the old model is not having suffix. 

Looks like there is a way to do it but I cannot find any clues or examples on how to use it. 
![Uploading image.png…]()

Thank you very much!

**Comment by cyclonstep** [comment]
the image:
![image](https://github.com/dbt-labs/dbt-core/assets/6695152/a55e6db1-28bf-4870-9cf2-e1ca67e8cbd3)

**Comment by jtcohen6** [comment, marked as answer]
@cyclonstep You can configure `yourmodelname_v1` with `alias: yourmodelname`. That way, it will not create a new object in the data warehouse, and continue to update the object located in the old namespace.

https://docs.getdbt.com/docs/collaborate/govern/model-versions#configuring-database-location-with-alias
